src/safeds/data/tabular/containers/_column.py

```diff
@@ -155,6 +156,11 @@ def plot(self) -> ColumnPlotter:
         """The plotter for the column."""
         return ColumnPlotter(self)
 
+    @property
+    def temporal(self) -> Temporal:
+        """Temporal operation for a column."""
+        return Temporal(self)
+
```

Comment:
Should only be available on cell. Columns can be transformed via the `transform` method.

Put the property next to https://github.com/Safe-DS/Library/blob/69a780cab1a5dcab33c8c7a36f03bd2eb3367683/src/safeds/data/tabular/containers/_cell.py#L145.

Maybe call it `dt`, like polars does. While we want to avoid abbreviations in general, these expressions can otherwise get very verbose.